docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -11,24 +11,19 @@ Using the command line Roslyn can be developed using the following pattern:
 
 ## Recommended version of .NET Framework
 
-The minimal required version of .NET Framework is 4.6, however 4.7.2 is recommended for best developer experience.
+The minimal required version of .NET Framework is 4.7.2.
 
-The projects in this repository are configured to build with Portable PDBs, which are supported in stack traces starting with .NET Framework 4.7.2.
-If a stack trace is displayed on .NET Framework older than 4.7.2 (e.g. by xUnit when a test fails) it won't contain source and line information.
+## Developing with Visual Studio 2019
```

Comment:
Is this being intentionally silent in saying nothing about 2017? I recognize it's not supported, but the lack of saying anything does say it's _not_ supported.

Reply:
When we were working on VS2017 we didn't say anything about VS2015 not being supported. If I have to list all the VS versions that arent' supported it will be rather long.

Reply:
"You must use 2019, everything else is unsupported" is short.